files/zh-cn/web/api/htmltemplateelement/content/index.md

```diff
@@ -5,7 +5,7 @@ slug: Web/API/HTMLTemplateElement/content
 
 {{APIRef("Web Components")}}
 
-The **`HTMLTemplateElement.content`**属性返回`<template>`元素的模板内容 (一个 {{domxref("DocumentFragment")}}).
+The **`HTMLTemplateElement.content`** 属性返回`<template>`元素的模板内容 (一个 {{domxref("DocumentFragment")}}).
```

Comment:
```suggestion
**`HTMLTemplateElement.content`** 属性返回 `<template>` 元素的模板内容（一个 {{domxref("DocumentFragment")}}）。
```

Reply:
emmm, it make sense

Reply:
I would fix it

Reply:
Note that @jasonren0403 also pointed out the modification suggestion for the halfwidth brackets, which is metioned [here](https://github.com/mdn/translated-content/blob/main/docs/zh-cn/translation-guide.md#%E6%A0%87%E7%82%B9%E7%AC%A6%E5%8F%B7).

Next time, if you are agreed with someone's suggestion, please click the Commit suggestion button to apply it (it's much faster and appropriate)

![image](https://user-images.githubusercontent.com/15844309/233524728-be8b5a18-0b27-400a-8d2f-28f4c9df84bf.png)

Reply:
Thanks, I got it